submissions/unabyband/popup-task/main.css

```diff
@@ -0,0 +1,302 @@
+    border: 2px solid rgba(0, 0, 0, 0.3);
+    border-radius: 10%;
+}
+
+.popup_window {
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+}
+
+.popup_window_list {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 28px 18px 23px 28px;
+}
+
+.popup_window_visible {
+    margin-top: 0px;
+    padding-top: 0px;
+    margin-bottom: 0px;
+}
+
+.popup_window_hidden {
+    display: none;
+    margin-top: 0px;
+}
+
+.popup_window__active {
+    text-decoration: none;
+    border: none;
+}
+
+.popup_window_list__item {
+    width: 80px;
+    height: 80px;
+    display: flex;
+    justify-content: center;
+    border: 1px solid transparent;
+    padding-top: 10px;
+}
+
+.popup_window_list__item:hover {
+    border: 1px solid rgba(0, 0, 0, 0.75); 
+}
+
+.popup_window__img {
+    height: 60px;
+    width: 60px;
+    display: inline-block;
+}
+.popup_window__img-src {
+    width: 100%;
+}
+.popup_window__label {
+    font-size: 13px;
+    text-align: center;
+    color: rgba(0, 0, 0, 0.87);
+    text-decoration: none;
+    border: none;
+    display: block;
+}
+
+.top_menu__item_other {
+    display: block;
+    flex: auto;
+    text-align: center;
+    margin-top: 10px;
+    padding-top: 0px;
+    margin-bottom: -35px;
+}
+.top_menu__item_other:hover {
+    text-decoration: underline;
+}
+
+.hidden_input {
+    appearance: none;
+    box-sizing:content-box;
+    margin: 0px;
+    border: none;
+}
+
+.hidden_input:focus ~ .morebutton {
+    border: 1px solid rgba(0, 0, 0, 0.75);
+    border-radius: 3%;
+}
+
+.morebutton {
+    display: block;
+    flex: auto;
+    text-align: center;
+    background-color: rgba(128, 128, 128, 0.1);
+    padding: 10px 0px 10px 0px;
+    margin-top: -22px;
+    cursor: pointer;
+    border: 1px solid transparent;
+}
+
+.hidden_input:checked ~ .popup_window_hidden {
+    border-top: 2px solid #e5e5e5;
+    display: flex;
+}
+
+.hidden_input:checked ~ .morebutton {
+    display: none;
+}
+
+.find {
+    display: flex;
+    justify-content: center;
+    padding: 50px 15px;
+}
+
+.find-container {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+}
+
+.find_cover {
+    width: 100%;
+    height: 100%;
+    padding-bottom: 30px;
+}
+
+.find_field {
+    width: 100%;
+    position: relative;
+}
+
+.find_field-input {
+    width: 100%;
+    height: 100%;
+    border: 0;
+    background: transparent;
+    outline: 0;
+    padding: 13px 45px 13px 30px;
+    box-sizing: border-box;
+    background: #F1F3F4;
+    border-radius: 22px;
+    transition: .2s;
+    color: #80868b;
+    font-size: 14px;
+}
+
+.find_field-icon {
+    width: 20px;
+    height: 30px;
+    position: absolute;
+    top: 5px;    
+    right: 15px;
+    background: url("../icons/mic.png") center no-repeat;
+    background-size: cover;
+}
+
+.find_field-input:focus {
+    border: 2px solid rgba(0, 0, 0, 0.75);
+}
+.find_field-input:hover {
+    background-color: rgba(0, 0, 0, 0.1);
+}
+
+@media screen and (max-width: 680px) {
```

Comment:
better to use common devices breakpoints 
https://www.lambdatest.com/blog/how-to-use-css-breakpoints-for-responsive-design/

Reply:
Done